src/revert_file.rs

```diff
@@ -68,6 +68,10 @@ impl RevertFile {
         Ok(())
     }
 
+    pub fn commit(mut self) {
+        self.reverted = true;
```

Comment:
This field is not read

Reply:
it gets read on drop

Reply:
Ah, you're right! I completely misread the patch...